packages/repo-tools/src/commands/repo/schema/openapi/lint.ts

```diff
@@ -47,6 +48,16 @@ async function lint(directoryPath: string, config?: { strict: boolean }) {
   const backstageRuleset = new Ruleset(
     {
       extends: [oas, ruleset],
+      rules: {
+        'allow-reserved-in-params': {
+          given: '$.paths..parameters[*]',
+          then: {
+            field: 'allowReserved',
+            function: truthy,
```

Comment:
An option here could be to lint for `allowReserved` being set, but allowing it to be `false`? I'd only want to do that if it's still very clear that `true` is the preferred value that should basically always be used though. Just thinking it can be nice to have an escape hatch if absolutely necessary, but kinda low prio tbh

Reply:
Hmm, yeah I think that's lower prio. The application can handle rejecting invalid pieces of the URL or incorrectly interpreting them, I don't think having it in the spec adds a bunch of value.